src/types.ts

```diff
@@ -41,10 +42,12 @@ export interface Attachment {
  */
 export interface ClientEvents {
 	channelCreate: [channel: AnyGuildChannel];
+	channelDelete: [channel: AnyGuildChannel];
 	channelUpdate: [channel: AnyGuildChannel];
 	error: [error: Error];
 	ready: [];
 	resumed: [];
+	unknown: [data: GatewayDispatchPayload];

```

Comment:
Doing `client.on("unknown", data => {})` seems very difficult to understand for me. I suggest renaming this event.
```suggestion
	unknownPayload: [data: GatewayDispatchPayload];
```